module/enrichers.mjs

```diff
@@ -489,6 +553,7 @@ async function enrichDamage(configs, label, options) {
     }
   }
   config.damageTypes = config.damageTypes.map(t => t.replace("/", "|"));
```

Comment:
Noticed a bug where this line throws an error if you don't specify damage type, e.g. `[[/damage 1d6]]`.